_workinggroups/softwaredevelopertools.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Software/Developer Tools 
+layout: plain
+---
+
+The Software/Developer Tools Working Group aims to help the HEP community in developing the latest, modern feature-rich projects
+with the aid of software tools such as code editors, static/dynamic code analyzers, compilers, debuggers, performance analyzers and so on.
+The group plans to accomplish this by mainly serving as a forum where developers can share their experiences, discuss new/existing tools they utilize,
+and learn from each other. To that point, contributions from the entire community are strongly encouraged.
+
+The group is currently coordinated by Giulio Eulisse (CERN, EP-AIP-SDS), Alaettin Serhan Mete (UC Irvine, ATLAS), and Martin Ritter (LMU, Belle II).
```

Comment:
One minor thing: I don't think LMU is as widely known as CERN so we should probably say "LMU Munich" to not confuse people. That said, I don't really mind either way.

Reply:
I just updated the text accordingly Martin, thanks.